<launch>

    <node pkg = "pcl_ros"
          type = "pcd_to_pointcloud"
          name = "pcd_to_pointcloud_node"
          args = "/mnt/WD500/OpenCV2021/experimentos/cloud3.pcd 1.0 _frame_id:=base_init"
          output = "screen">
    </node>

    <node pkg = "espeleo_modifications"
          type = "file_to_path_single.py"
          name = "file_to_path_node"
          args = "_path:=/mnt/WD500/OpenCV2021/experimentos/rtab_odom.txt"
          output = "screen">
    </node>

    <node pkg = "rviz"
          type = "rviz"
          name = "rviz_node"
          args = "-d /mnt/WD500/OpenCV2021/experimentos/slam.rviz"
          output = "screen">  
    </node>

</launch>
